lib/mimdb_web/controllers/movie_controller.ex

```diff
@@ -0,0 +1,69 @@
+defmodule MimdbWeb.MovieController do
+  use MimdbWeb, :controller
+
+  alias Mimdb.Movies
+  alias Mimdb.Movies.Movie
+
+  def index(conn, _params) do
+    movies = Movies.list_movies()
+    render(conn, "index.html", movies: movies)
+  end
+
+  def new(conn, _params) do
+    changeset = Movies.change_movie(%Movie{})
+    genres = Movies.list_genres()
+    render(conn, "new.html", changeset: changeset, genres: genres)
+  end
+
+  def create(conn, %{"movie" => movie_params}) do
+    genres = Movies.list_genres()
```

Comment:
This can be moved into error branch so that we don't have to make the added query on happy path